pandas/tests/test_algos.py

```diff
@@ -254,6 +254,21 @@ def test_value_counts_nat(self):
         tm.assert_series_equal(algos.value_counts(dt), exp_dt)
         # TODO same for (timedelta)
 
+    def test_dropna(self):
+        tm.assert_series_equal(
```

Comment:
pls add a test that has nan in it as well (for these dtypes too, e.g. bool, float), even though adding nan will make the bool -> object